src/main/java/org/openapijsonschematools/codegen/generators/DefaultGenerator.java

```diff
@@ -1368,6 +1373,10 @@ protected void updateModelForComposedSchema(CodegenSchema m, Schema schema, Stri
         }
     }
 
+    public LinkedHashSet<String> getOauthServerHostnames() {
```

Comment:
How about returning null if oauthServerHostnames is empty?
Then you can simplify the call site to:
`bundle.put("oauthServerHostnames", generator.getOauthServerHostnames());`

Reply:
Could you provide me more details about this? I do not get it completely.

Reply:
You can update the code here to have:
```
if (oauthServerHostnames.isEmpty()) {
    return null;
}
```
Then change the caller to remove the if statement and use `bundle.put("oauthServerHostnames", generator.getOauthServerHostnames());`